### DISCUSSION TITLE
even when setting the exe, streamlink says executeable can't be found.

### DISCUSSION THREAD

**Discussion by MonthOLDpickle** [streamlink/streamlink-twitch-gui#1025] [Q&A] [answered]
I point it to the same exe I use to sue VLC but I get the error saying it can't find it.

Windows 11. What am I doing wrong?

**Comment by bastimeyer** [comment]
It's not even clear from your description what you're doing? Do you have issues with Streamlink or your player? Post an actual error message and an exact description of what you're trying to do. Otherwise, read the wiki.

1. https://github.com/streamlink/streamlink-twitch-gui/wiki/Installation
2. https://github.com/streamlink/streamlink-twitch-gui/wiki/Streamlink-configuration
3. https://github.com/streamlink/streamlink-twitch-gui/wiki/Player-configuration

**Comment by MonthOLDpickle** [comment]
The error message is basically missing executable. It can't find it. Even in settings its pointing to vlc.exe (where I told it is) and the option to use VLC is selected. 


VLC runs fine for everything.

**Reply by bastimeyer** [reply, marked as answer]
You are not answering my questions.

Just as a reminder, since I think you're missing something here: You are using **Streamlink Twitch GUI**, which is a launcher for **Streamlink**, which itself launches the player (VLC in your case), configured by **Streamlink Twitch GUI**.

There are different "Missing executable" errors that can be shown, depending on your mis-configuration. This is btw labeled clearly, so as said, you're not describing the full issue.

Like I've already told you, please **carefully read and follow the setup and configuration instructions from the wiki** if you can't manage to get this working on your own. That should be absolutely trivial though, unless you don't understand the relation between Streamlink Twitch GUI, Streamlink, and your video-player. To me, this sounds like you didn't install Streamlink, so Streamlink Twitch GUI can't find the Streamlink executable.

**Reply by MonthOLDpickle** [reply]
So..then do I redownload  streamlink-6.11.0-1-py312-x86_64.exe  or  streamlink-6.11.0-1-py38-x86_64.exe  cause that may be my issue.  
  
When I go into documentation all it talks is about additional commands you can do.

**Reply by bastimeyer** [reply]
https://github.com/streamlink/windows-builds/tree/6.11.0-1?tab=readme-ov-file#streamlinkwindows-builds

**Reply by MonthOLDpickle** [reply]
That is where I got both these exes which one is the one I want 6.11.0-1-py38-x86_64.exe or 6.11.0-1-py312-x86_64.exe

**Reply by mkbloke** [reply]
And what does it say on the page bastimeyer linked to above, under flavors [sic] and recommended?

**Reply by MonthOLDpickle** [reply]
Says nothing of either py38 or py312, like which python one should be used. I am already using the rec x86_x64  
  
Kept on reinstalling the 3.12 one (cause I think the is newer than 3.8 according to py page) and it finally worked.
